ga4gh/serverconfig.py

```diff
@@ -84,3 +86,29 @@ class TestOidcConfig(TestConfig):
     OIDC_AUTHZ_ENDPOINT = "https://accounts.example.com/auth"
     OIDC_TOKEN_ENDPOINT = "https://accounts.example.com/token"
     OIDC_TOKEN_REV_ENDPOINT = "https://accounts.example.com/revoke"
+
+
+class FlaskDefaultConfig(object):
```

Comment:
I don't think so.  "Modifying" flask for us usually means modifying the config, which is just, essentially, a dictionary.  Reseting means getting this dictionary back to the defaults.  The Flask API does not seem to provide any method to do this for us: http://flask.pocoo.org/docs/0.10/api/#flask.Config